source/MaterialXCore/Version.cpp

```diff
@@ -578,6 +578,7 @@ void Document::upgradeVersion()
     {
         // Convert color2 types to vector2
         const StringMap COLOR2_CHANNEL_MAP = { { "r", "x" }, { "a", "y" } };
+        const string CHANNELS_ATTRIBUTE_STRING = "channels";
```

Comment:
Put in a const string since it's repeated in a number of places.

Reply:
In this case, it might be clearer to use the string literals, so that readers remain aware that these constants don't actually exist in the MaterialX codebase.  That's the pattern we've been using elsewhere in the version upgrade system, especially in more recently-contributed changes.